homeassistant/components/zwave_js/binary_sensor.py

```diff
@@ -248,6 +248,27 @@ class PropertyZWaveJSEntityDescription(BinarySensorEntityDescription):
 }
 
 
+@callback
+def is_valid_binary_notification_sensor(
```

Comment:
I'd rename this function since it returns an entity description and not a boolean.

Reply:
I actually changed it back as it was causing a side effect. The check is now separated from the entity description